apps/laboratory/src/pages/library/multichain-basic.tsx

```diff
@@ -2,13 +2,15 @@ import { createAppKit } from '@reown/appkit/react'
 import { ThemeStore } from '../../utils/StoreUtil'
 import { ConstantsUtil } from '../../utils/ConstantsUtil'
 import { AppKitButtons } from '../../components/AppKitButtons'
-import { mainnet, optimism, arbitrum, solana } from '@reown/appkit/networks'
+import { mainnet } from '@reown/appkit/networks'
 import { MultiChainInfo } from '../../components/MultiChainInfo'
-
 import { UpaTests } from '../../components/UPA/UpaTests'
 
+const networks = [...ConstantsUtil.EvmNetworks, ...ConstantsUtil.SolanaNetworks]
```

Comment:
Maybe something like 
```suggestion
const networks = ConstantsUtil.AllNetworks
```

Low prio tho